docs/resources/Stage_Instance.md

```diff
@@ -0,0 +1,70 @@
+# Stage Instance Resource
+
+A _Stage Instance_ holds information about a live stage.
+
+## Definitions
+
+Below are some definitions related to stages.
+
+- **Liveness:** A Stage channel is considered _live_ when there is an associated stage instance. Conversely, a Stage channel is _not live_ when there is no associated stage instance.
+- **Speakers:** A participant of a Stage channel is a _speaker_ when their [voice state](#DOCS_RESOURCES_VOICE/voice-state-object)
+  is not `suppress`ed, and has no `request_to_speak_timestamp`.
+- **Moderators**: A member of the guild is a _moderator_ of a Stage channel if they have all of the following [permissions](#DOCS_TOPICS_PERMISSIONS/permissions):
+  - `MANAGE_CHANNELS`
+  - `MUTE_MEMBERS`
+  - `MOVE_MEMBERS`
+
+## Auto Closing
+
+When a Stage channel has no speakers for a certain period of time (on the order of minutes) it will be automatically deleted.
+
+###### Stage Instance Structure
+
+| Field      | Type      | Description                                  |
+| ---------- | --------- | -------------------------------------------- |
+| id         | snowflake | The id of this Stage instance                |
+| guild_id   | snowflake | The guild id of the associated Stage channel |
+| channel_id | snowflake | The id of the associated Stage channel       |
+| topic      | string    | The topic of the Stage (1-120 characters)    |
```

Comment:
`topic` here is for the stage instance, or the stage itself?

Reply:
Ah. I would probably say Stage instance.

Regardless, it will be what controls the topic text here now (and in other places).

![image](https://user-images.githubusercontent.com/5456182/117551041-80a55080-aff8-11eb-95e4-b486cfa128ef.png)